### PR TITLE
absolute path to load owl file

### PR DESCRIPTION
**Pull request by chcorbato**
to address https://github.com/rosin-project/metacontrol_experiments/issues/16

### CODE REVIEW

**Comment by chcorbato** [comment]
I have created the branch 'melodic' to follow the developments in the branch with the same name in mc_mros_reasoner

File: run_reasoner.sh [hunk 1]
```diff
@@ -151,7 +151,7 @@ echo "Launching: mros reasoner"
 gnome-terminal --window --geometry=80x24+10+10 -- bash -c "source $PYTHON3_VENV_PATH/venv3.6_ros/bin/activate;
 source $PYTHON3_VENV_PATH/devel/setup.bash;
 source $REASONER_WS_PATH/devel/setup.bash;
-roslaunch mros1_reasoner run.launch onto:=kb.owl;
+roslaunch mros1_reasoner run.launch onto:=$REASONER_WS_PATH/src/mros1_reasoner/mros1_reasoner/scripts/kb.owl;
```
Comment (gavanderhoorn) [inline run_reasoner.sh:154]: Why not use `$(rospack find <your_pkg_name>)` instead of some env variable?

Comment (chcorbato) [reply run_reasoner.sh:154]: Good point! But can we use that in the shell script?
@marioney please consider that

Comment (gavanderhoorn) [reply run_reasoner.sh:154]: `rospack` is a regular Python script, not a `roslaunch` substitution command.

Comment (chcorbato) [reply run_reasoner.sh:154]: I should follow Robotic Software Practicals...  :-)

**Comment by marioney** [comment]
This merge is no longer needed as #19 overwrites it, and actually uses the `$(rospack find)`

**Pull request closed without merging**